### PR TITLE
Add CI support and basic test structure

### PR DESCRIPTION
**Pull request by hallfox**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+  - "3.5"
+install:
+  - pip install -r requirements.txt
+script: nosetests
```
Comment (jpalazz2) [inline .travis.yml:6]: This is where we would add our own custom scripts to test stuff?

Comment (hallfox) [inline .travis.yml:6]: We may also want to run linter checks just to be picky

Comment (hallfox) [inline .travis.yml:6]: I personally use flake8

Comment (jpalazz2) [inline .travis.yml:6]: Pick a linter you prefer and add that. Document it somewhere and we can get people to switch to it.

**Pull request merged**